core-bundle/src/Resources/contao/dca/tl_page.php

```diff
@@ -544,8 +544,8 @@ static function ($value)
 		(
 			'exclude'                 => true,
 			'inputType'               => 'select',
-			'options'                 => array(0=>'http://', 1=>'https://'),
-			'eval'                    => array('tl_class'=>'w50'),
+			'options'                 => array('http://', 'https://'),
+			'eval'                    => array('tl_class'=>'w50', 'isAssociative' => true),
```

Comment:
Why the flag?

Reply:
See https://github.com/contao/contao/pull/4797#discussion_r909607112 - otherwise the select won't work.